lib/ecb_rates/application.rb

```diff
@@ -0,0 +1,28 @@
+require 'date'
+
+module EcbRates
+  class Application
```

Comment:
Have you consider to move content of `class Application` right to module EcbRates like [here](https://github.com/jnicklas/capybara/blob/master/lib/capybara.rb)?